sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/TimeSeriesPoint.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Represents a Time Series point at a specific timestamp.
+    /// </summary>
+    public struct TimeSeriesPoint
+    {
+        private readonly IDictionary<string, PropertyValues> _propertyNameToPageValues;
```

Comment:
Why is it called PageValues and not PropertyValues?

Reply:
I could change the name. I guess the main reason is that this TimeSeriesPoint points to the values at this page only, and not all property values across all pages.

Reply:
Got it. It can use a comment in either case, with pageValues (property Values in this page only). If you are renaming then what you mentioned above.